src/pages/Home.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+import ScopedCssBaseline from '@mui/material/ScopedCssBaseline';
+
+export const Home = () => {
+  const navigate = useNavigate();
+
+  const handleLoginClick = () => {
+    navigate('/login');
+  };
+
+  const handleRegisterClick = () => {
+    navigate('/register');
+  };
+  return (
+    <ScopedCssBaseline>
+      <h2>Home</h2>
+      <button onClick={handleRegisterClick}>Register</button>
+      <button onClick={handleLoginClick}>Login</button>
+      <ul>
+        <li>
+          <Link to="/UserDashboard">User Dashboard</Link>
+        </li>
+        <li>
+          <Link to="/AdminDashboard">Admin Dashboard</Link>
```

Comment:
same here

Reply:
oh btw, dashboards is gonna be similar, or even if know they should be under the same path, but the different things would be displayed for each user